consensus/hotstuff/committees/consensus_committee_test.go

```diff
@@ -139,7 +139,7 @@ func TestConsensus_LeaderForView(t *testing.T) {
 		})
 
 		t.Run("after current epoch", func(t *testing.T) {
-			t.SkipNow()
+			unittest.SkipUnless(t, unittest.TEST_TODO, "temporarily disabled")
```

Comment:
```suggestion
			unittest.SkipUnless(t, unittest. TEST_WIP, "disabled as the current implementation uses a temporary fallback measure in this case (triggers EECC), rather than returning an error")
```